### PR TITLE
Add basic device schema based on UI

### PR DESCRIPTION
**Pull request by chuckcarpenter**
**What kind of change does this PR introduce?**

Adds the basic device schema from fields documented in Lantern and used in the Torch UI.

**Does this PR introduce a breaking change?**

No, new Zeus schema.

### CODE REVIEW

**Comment by chuckcarpenter** [comment]
@shanewilson trying to be ruthless in the fields as our previous PR to get this in as needed. Let me know if there are any obvious omissions.

File: src/services/Device/schema.graphql [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  cursor: ID!
+  node: DeviceProfile!
+}
+
+type DeviceRelayConnection {
+  edges: [DeviceEdge!]!
+  pageInfo: ConnectionPageInfo!
+}
+
+type UserProfileRelayConnection {
```
Comment (tazsingh) [inline src/services/Device/schema.graphql]: Looks like this isn't needed?

File: src/services/Device/schema.graphql [hunk 2]
```diff
@@ -0,0 +1,42 @@
+interface Node {
+  id: ID!
+}
+
+extend type Query {
+  devices(after: ID, before: ID, first: Int, last: Int): DeviceRelayConnection!
+}
+
```
Comment (MarcelCutts) [inline src/services/Device/schema.graphql]: We'll need to add mutations for device association too

File: src/services/Device/schema.graphql [hunk 3]
```diff
@@ -0,0 +1,42 @@
+interface Node {
+  id: ID!
+}
+
+extend type Query {
+  devices(after: ID, before: ID, first: Int, last: Int): DeviceRelayConnection!
```
Comment (tazsingh) [inline src/services/Device/schema.graphql]:
```suggestion
  deviceModels(nameFilter: String, after: ID, before: ID, first: Int, last: Int): DeviceModelRelayConnection!
```

**Pull request merged**